modules/demo-shopware/cypress/integration/category.js

```diff
@@ -1,27 +1,24 @@
-import Base from '~/cypress/support/pageObjects/Base'
-import Category from '~/cypress/support/pageObjects/Category'
+import visitCategory from '~/cypress/support/pageObjects/category/visitCategory'
+import continueShopping from '~/cypress/support/pageObjects/base/continueShopping'
+import expectMicrocartQuantity from '~/cypress/support/pageObjects/base/expectMicrocartQuantity'
+import getCategoryTitle from '~/cypress/support/pageObjects/base/getCategoryTitle'
+
 import Listing from '~/cypress/support/pageObjects/Listing'
 
 describe('Category', () => {
-  /** @type {Base} */
-  let base
-
-  /** @type {Category} */
-  let category
-
   /** @type {Listing} */
   let listing
 
   beforeEach(() => {
-    base = new Base()
-    category = new Category()
     listing = new Listing()
 
-    category.visit('Office')
+    visitCategory('Office').as('category')
   })
 
   it('contains category title', () => {
-    category.getTitle().will('include.text', () => category.name)
+    cy.get('@category').then((category) => {
+      getCategoryTitle().will('include.text', () => category.name)
```

Comment:
I don't think will is needed here anymore because the category is already properly evaluated when this line gets executed.